lidar_multiclass/data/loading.py

```diff
@@ -97,22 +106,27 @@ def split_and_save(self, filepath: str, output_subdir_path: str) -> None:
         Args:
             filepath (str): input LAS file
             output_subdir_path (str): output directory to save splitted `.data` objects.
+
         """
+        range_by_axis = np.arange(
+            self.input_tile_width_meters // self.subtile_width_meters + 1
+        )
+
         data = self.load_las(filepath)
         idx = 0
-        for _ in tqdm(self.range_by_axis):
+        for _ in tqdm(range_by_axis):
             if len(data.pos) == 0:
```

Comment:
Je ne connais pas bien le type Data, mais si c'est comme le reste de python peut-être que : 
if not data.pos:
fait la même chose que : 
if len(data.pos) == 0: 
d'une manière plus lisible

Reply:
En fait `data.pos` est un np.ndarray, ce qui l'expose à 
`ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()`.
J'ai cependant simplifier par un "if not len(data.pos)" dirctement.